src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -106,8 +111,39 @@ public void createDataset(Dataset dataset) {
 
             bigQueryProject.createDataset(datasetName, dataset.getDescription());
             for (DatasetTable table : dataset.getTables()) {
-                bigQueryProject.createTable(datasetName, table.getRawTableName(), buildSchema(table, true));
-                bigQueryProject.createTable(datasetName, table.getSoftDeleteTableName(), buildSoftDeletesSchema());
+                PartitionStrategy partitionStrategy = table.getPartitionStrategy();
+                Column partitionColumn = table.getPartitionColumn();
+                Schema rawSchema = buildSchema(table, true);
+
+                if (partitionStrategy == null) {
```

Comment:
Should we always have a partitionStrategy; e.g., NONE. Even if it isn't in the database, it would allow a switch across types or a class structure instead of this logic flow.